src/io/flutter/logging/FlutterLog.java

```diff
@@ -57,9 +57,9 @@ public void clear() {
     // TODO(pq): add locking.
     entries.clear();
   }
-
+  
   public List<FlutterLogEntry> getEntries() {
-    return Collections.unmodifiableList(entries);
+    return Collections.unmodifiableList(new ArrayList<>(entries));
```

Comment:
You could use also `ImmutableList.copyOf()` from Guava.

Reply:
Nice.  Good idea!